docs/user-guides/deploying.md

```diff
@@ -28,7 +28,7 @@ clone this repository:
 $ git clone https://github.com/jetstack/cert-manager
 $ cd cert-manager
 # check out the latest release tag to ensure we use a supported version of cert-manager
-$ git checkout v0.2.3
+$ git checkout -b v0.2.3
```

Comment:
The `-b` flag on `git checkout` will cause git to create a new branch if it does not already exist, which isn't the behaviour we want here.

Can you elaborate the issue you were running into that caused you to add this flag?

See:
```
git checkout -b|-B <new_branch> [<start point>]
           Specifying -b causes a new branch to be created as if git-branch(1) were called and then checked out.
```

Reply:
Ok, sorry.